packages/front-end/services/track.ts

```diff
@@ -81,15 +81,21 @@ const SESSION_ID_COOKIE = "gb_session_id";
 const pageIds: Record<string, string> = {};
 
 const dataWareHouseTrack = async (event: DataWarehouseTrackedEvent) => {
-  if (!dataWarehouseUrl) return;
+  if (inTelemetryDebugMode()) {
+    console.log("Telemetry Event - ", event);
+  }
+  if (!isTelemetryEnabled()) return;
```

Comment:
This line means we no longer send the event when in debug mode. I think we still want to send the telemetry as we are debugging, at we are trying to output failed requests in debug mode below on line 101.